spark-jobs/src/main/scala/filodb/cardbuster/PerShardCardinalityBuster.scala

```diff
@@ -83,12 +82,13 @@ class PerShardCardinalityBuster(dsSettings: DownsamplerSettings,
         }
       }
       if (willDelete) {
-        BusterContext.log.debug(s"Deleting part key $pkPairs from shard=$shard")
+        BusterContext.log.info(s"Deleting part key $pkPairs from shard=$shard startTime=${pk.startTime} " +
+          s"endTime=${pk.endTime} split=$split isSimulation=$isSimulation")
       }
       willDelete
     }.foreach { pk =>
       try {
-        colStore.deletePartKeyNoAsync(dataset, shard, pk)
+        if (!isSimulation) colStore.deletePartKeyNoAsync(dataset, shard, pk.partKey)
         numPartKeysDeleted.increment()
```

Comment:
Do we need to increment this counter for simulation?

Reply:
It'd be nice for us to see the metrics in dashboard even during simulation

Reply:
Ok, it might be misleading in the dashboard when someone views the metrics whether it was simulation or real deletion, perhaps a label stating runType might be useful?